diff --git a/v4l/compat.h b/v4l/compat.h
index 51cdae8..47ea96b 100644
--- a/v4l/compat.h
+++ b/v4l/compat.h
@@ -670,7 +670,7 @@ static inline long get_user_pages_longterm(unsigned long start,
                 unsigned long nr_pages, unsigned int gup_flags,
                 struct page **pages, struct vm_area_struct **vmas)
 {
-        return get_user_pages(start, nr_pages, gup_flags, pages, vmas);
+        return get_user_pages(start, nr_pages, gup_flags, pages);
 }
 #endif
 #endif
