diff --git a/v4l/Makefile b/v4l/Makefile
index d6e4d69..1cc001f 100644
--- a/v4l/Makefile
+++ b/v4l/Makefile
@@ -192,7 +192,7 @@ ifeq ($(inst-m),)
   inst-m	:= $(obj-m)
 endif
 
-v4l_modules := $(shell /sbin/lsmod|cut -d' ' -f1 ) $(patsubst %.ko,%,$(inst-m))
+v4l_modules := $(shell /bin/lsmod|cut -d' ' -f1 ) $(patsubst %.ko,%,$(inst-m))
 
 #################################################
 # locales seem to cause trouble sometimes.
